cmd/version/version_test.go

```diff
@@ -27,11 +27,11 @@ func TestVersionWorksWithoutAccessibleConfigFile(t *testing.T) {
 	// re-wire
 	oldOsStdout := os.Stdout
 	oldConfigPath := config.GetConfigPath()
-	_ = config.SetConfigPath(path)
+	assert.NoError(t, config.SetConfigPath(path))
```

Comment:
These changes should have gone in the previous commit unless you are planning to squash them?

Reply:
Ai... Probably was a bit too quick on the trigger with "git commit --amend --no-edit". Will fix!